### DISCUSSION TITLE
What's the best way to bulk require files in node on Zeit/Next?

### DISCUSSION THREAD

**Discussion by techwraith** [vercel/vercel#4076] [General]
I have a directory full of small modules that all take the same arguments and return the same format. I would like to use something like bulk-require (https://www.npmjs.com/package/bulk-require) to pull them all in to one of my API routes in my Next application.
Is this possible? What's the best way to achieve this?

**Comment by coetry** [comment]
@techwraith I'm not sure if I fully understand the question. Did you try doing this and run into an error? It would also help if you had a minimal repo that shows the pattern you're describing.

**Reply by techwraith** [reply]
Ah, we haven't tried it yet, I'm just not sure if `bulk-require` is supported (it does some magic under the hood).

To more clearly ask:

What is the best/recommended way to bulk require in Next/Now?
